controllers/getArticlesController.js

```diff
@@ -0,0 +1,31 @@
+const {
+  getArticleById,
+  getArticlesModels,
+} = require("../models/getArticlesModel");
+
+async function getArticleController(req, res, next) {
+  try {
+    const returnedArticles = await getArticlesModels();
+
+    res.status(200).send({ articles: returnedArticles });
+  } catch (error) {
+    next(error);
+  }
+}
```

Comment:
This is ticket 5, the current PR is ticket four. 

Please take this out this PR - there doesnt seem to be any tests for this either

Reply:
Done